src/test/java/core/basesyntax/service/impl/FileWriterImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.service.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class FileWriterImplTest {
+    private static final String HEADER = "fruit,quantity";
+    private FileWriter fileWriter;
+    private String report;
+
+    @Before
+    public void setUp() {
+        report = System.lineSeparator() + "banana,152" + System.lineSeparator() + "apple,90";
+        fileWriter = new FileWriterImpl();
+    }
+
+    @Test
+    public void writeReport_ok() {
+        List<String> expected = new ArrayList<>();
+        expected.add("fruit,quantity");
+        expected.add("banana,152");
+        expected.add("apple,90");
+        fileWriter.writeReport("src/main/resources/report.csv", report, HEADER);
+        try {
+            Path filePath = Path.of("src/main/resources/report.csv");
+            List<String> actual = Files.readAllLines(filePath);
+            assertEquals(actual, expected);
+            Files.delete(filePath);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from report file");
```

Comment:
```suggestion
            throw new RuntimeException("Can't read data from report file");
```

lets make exception messages more informative and add file path to it